app/src/main/java/mozilla/lockbox/action/SettingAction.kt

```diff
@@ -20,30 +20,20 @@ open class SettingAction : Action {
 }
 
 class Setting {
-    enum class AutoLockTime {
-        OneMinute,
-        FiveMinutes,
-        FifteenMinutes,
-        ThirtyMinutes,
-        OneHour,
-        TwelveHours,
-        TwentyFourHours,
-        Never;
+    enum class AutoLockTime(
+        @StringRes val stringValue: Int,
+        private val seconds: Long
+    ) {
+        OneMinute(R.string.one_minute, 60),
+        FiveMinutes(R.string.five_minutes, 60 * 5),
+        FifteenMinutes(R.string.fifteen_minutes, 60 * 15),
+        ThirtyMinutes(R.string.thirty_minutes, 60 * 30),
+        OneHour(R.string.one_hour, 60 * 60),
+        TwelveHours(R.string.twelve_hours, 60 * 60 * 12),
+        TwentyFourHours(R.string.twenty_four_hours, 60 * 60 * 24),
+        Never(R.string.never, 0);
```

Comment:
Assuming we eventually support "Lock Immediately", what might its `seconds` be?  `-1`?

I don't know that it's worth even considering right now, but I hope eventually 😄

Reply:
tbh I don't think it would be that hard; we'd just lock on every foreground without checking the stored time at all if that was the user setting :p

Reply:
touché (-: